### ISSUE TITLE
Shortlist of important backlogged Android Issues

### ISSUE THREAD

**Issue by perpetuatingcuriosity** [creativecommons/list#328]
Essentials:
- #164 Tapping list item w/ upload error should bring up dialog to retry (waiting on #163, #173)
- #234 Longer item descriptions are cut off in RandomActivity
- #299 Ask user to upgrade if the app is not the current version and they are not using play store (waiting on #299 api component)
- #325 Update API requests to retrofit

General layout fixes (review for phablet sizes screens), including:
- #326 Landscape mode for screens at all sizes using savedInstanceState (start with popular sizes)
- #324 Ensure layouts support all screen sizes

Nice-to-haves
- #269/#296  Show attribution in the app, likely through a “view my license” page and/or user history feed (connected to #295: Users should be able to see items they’ve already uploaded)
- #306 During onboarding, overlay explanation for adding item suggestions to your list
